iceoryx_posh/test/moduletests/test_popo_condition_variable.cpp

```diff
@@ -99,94 +86,6 @@ TEST_F(ConditionVariable_test, NoNotifyResultsInNotBeingTriggered)
     EXPECT_FALSE(m_waiter.wasNotified());
 }
 
-TEST_F(ConditionVariable_test, WasTriggerCallDoesNotChangeTheState)
```

Comment:
I guess this tests are removed because it tests the functionality of a member which already is tested, right?

Reply:
Exactly. @FerdinandSpitzschnueffler tested all this stuff in a very clean fashion already in the event variable. Here we merged the event variable tests into condition variable and the old tests are removed. For instance `NotifyOnceResultsInBeingTriggered` tests exactly this.